yaml/pois.yaml

```diff
@@ -414,3 +414,45 @@ filters:
     min_zoom: 15
     output:
       kind: {col: tags->whitewater}
+  - filter: {aerialway: pylon}
```

Comment:
Oh, no, I got confused about the `pylon` - that's not for electricity but for holding up aerialway cables. Still, is `pois` the appropriate place for this? I am sure that many people _are_ interested in pylons, but perhaps it's not traditionally considered a point of interest.